README.md

```diff
@@ -117,165 +109,129 @@ mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-rest.jar -DpomFil
 
 ### Kubernetes Prerequisites
 
-Additionally if you wish to run the application in Kubernetes using the `Coherence Operator`, you must
-ensure you carry out the following:
-
-* Check Software and Runtime Prerequisites
-
-  Ensure you meet the `Software and Runtime Prerequisites` in the 
-  [Oracle Coherence Operator Quick Start Guide](https://oracle.github.io/coherence-operator/docs/quickstart.html#prerequisites).
+To run the application in Kubernetes using the Coherence Operator, you must
+ensure the following requirements:
 
-* Add the Helm repository 
+* Check [Software Requirements](https://github.com/oracle/coherence-operator/blob/gh-pages/docs/quickstart.md#software-requirements) and [Runtime Environment Requirements](https://github.com/oracle/coherence-operator/blob/gh-pages/docs/quickstart.md#runtime-environment-requirements) in the *Quick Start Guide*.
 
-   Issue the following to create a `coherence` helm repository:
+* Add the Helm repository. Execute the following commands to create a `coherence` helm repository:
 
-   ```bash
+  ```bash
    $ helm repo add coherence https://oracle.github.io/coherence-operator/charts
-
    $ helm repo update
-
-   Hang tight while we grab the latest from your chart repositories...
-   ...Skip local chart repository
-   ...Successfully got an update from the "coherence" chart repository
    ```
 
-* Ensure you can pull the Coherence Docker Image	
-
-  1. Go to to [Oracle Container Registry](https://container-registry.oracle.com)
-
-  1. Search for "Coherence".	
-
-  1. Select `coherence` from the list.	
-
-  1. Click on `Sign-in` on the right and enter your credentials, or create and account if you don't already have one.	
+### Pull Coherence Docker Image
```

Comment:
We need to rename this as we are not actually pulling the docker image, we are ensuring we can pull the image by accepting the terms, conditions, etc.